lib/components/atoms/Blockquote/Blockquote.story.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import defaultConfig from './Blockquote.mock';
+
+// Import Styled Component to showcase variations
+import Blockquote, { BlockquoteVanilla } from '.';
+
+// TODO: indentation is not working fine in editor. Need to check for .hbs files
+// $FlowFixMe eslint-disable-next-line extra-rules/no-commented-out-code
```

Comment:
Remove this // $FlowFixMe eslint-disable-next-line extra-rules/no-commented-out-code

Reply:
fixed